src/QsFmt/Formatter.Tests/Examples.fs

```diff
@@ -78,3 +78,59 @@ let ``Removes extraneous spaces`` =
         return x w/ Foo <- (7, y);
     }
 }"""
+
+[<Example(ExampleKind.Update)>]
+let ``Updates Using Statements`` =
+    """namespace Foo {
+    operation Bar() : Unit {
+        using ((qubits, q) = (Qubit[2], Qubit())) {
+            using (q2 = Qubit());
+        }
+    }
+}""",
+
+    """namespace Foo {
+    operation Bar() : Unit {
+        use (qubits, q) = (Qubit[2], Qubit()) {
+            use q2 = Qubit();
+        }
+    }
+}"""
+
+[<Example(ExampleKind.Update)>]
+let ``Updates Borrowing Statements`` =
+    """namespace Foo {
+    operation Bar() : Unit {
+        borrowing ((qubits, q) = (Qubit[2], Qubit())) {
+            borrowing (q2 = Qubit());
+        }
+    }
+}""",
+
+    """namespace Foo {
+    operation Bar() : Unit {
+        borrow (qubits, q) = (Qubit[2], Qubit()) {
+            borrow q2 = Qubit();
+        }
+    }
+}"""
+
+[<Example(ExampleKind.Update)>]
```

Comment:
This test passes its intended test for checking the `update` of the `use` syntax, but fails because it is also used in the FixedPoint tests. I'm becoming more and more convinced that this testing infrastructure is too complex and needs to be flattened into simple unit tests.

Reply:
In `FixedPointData`, try filtering `examples` to only include those with kind `ExampleKind.Format`.

https://github.com/microsoft/qsharp-compiler/blob/0c595bb579ed42be8087557be26220147df650b5/src/QsFmt/Formatter.Tests/Discoverer.fs#L177

Reply:
Or, actually, `Example.toFixedPoint` should probably return an option now. Only format examples can be converted to a fixed point, so update examples can return `None`. Then `FixedPointData` can use `Seq.choose`.

Reply:
Ah okay that fixed it, thanks!